azure-pipelines.yml

```diff
@@ -47,71 +47,3 @@ jobs:
   - publish: _build/log
     artifact: Dune Build Log
     condition: always()
-
-- job: macOS
-  pool:
-    vmImage: 'macOS-10.14'
-
-  variables:
-    MACOSX_DEPLOYMENT_TARGET: '10.11'
-
-  steps:
-
-  - checkout: self
-    fetchDepth: 10
-
-  - script: |
-      set -e
-      export export HOMEBREW_NO_INSTALL_CLEANUP=1
```

Comment:
I didn't keep this recently introduced line because I didn't (yet) observe a similar problem on GitHub Actions. But we could decide to keep it, of course. BTW @gares, what is the point of the double `export`?